tests/DfE.FindInformationAcademiesTrusts.UnitTests/Pages/SearchModelTests.cs

```diff
@@ -0,0 +1,43 @@
+using DfE.FindInformationAcademiesTrusts.Pages;
+
+namespace DfE.FindInformationAcademiesTrusts.UnitTests.Pages;
+
+public class SearchModelTests
+{
+    [Fact]
+    public async Task OnGetAsync_should_search_if_query_parameter()
+    {
+        var query = "trust";
+        var mockTrustSearch = new Mock<ITrustSearch>();
+        mockTrustSearch.Setup(s => s.SearchAsync(
+            It.Is<string>(q => q == query)
+        ).Result).Returns(
+            new[]
+            {
+                new Trust("trust 1"),
+                new Trust("trust 2"),
+                new Trust("trust 3")
+            }
+        ).Verifiable();
```

Comment:
No need for `Verifiable()` as we are not directly checking that the moq object has been called